language/tools/move-cli/src/base/movey_upload.rs

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::utils::{credential, credential::TestMode};
+use anyhow::bail;
+use clap::*;
+use move_package::{resolution::resolution_graph::MOVEY_URL, BuildConfig};
```

Comment:
I think it makes more sense to put the `MOVEY_URL` variable in this file instead.